src/booleans.js

```diff
@@ -1,62 +1,98 @@
-function negate(a) {
-  // your code here
-};
+const negate = (a) => !a;
 
-function both(a, b) {
-  // your code here
-};
+const both = (a, b) => {
+  if (!a || !b) {
+    return false;
+  }
+  return true;
+}
 
-function either(a, b) {
-  // your code here
-};
+const either = (a, b) => {
+  if (a || b) {
+    return true;
+  }
+  return false;
+}
+const none = (a, b) => {
+  if (a || b) {
```

Comment:
I like this because it is a slightly different solution to what I expected, but it still works. I would use `return !a && !b` here.